### PR TITLE
feat: add known issue about upgrade from v1.3.2 to v1.4.0

### PR DESCRIPTION
**Pull request by FrankYang0529**
https://github.com/harvester/harvester/issues/7088

### CODE REVIEW

**Review by ibrokethecloud — APPROVED** [review]

**Review by bk201 — APPROVED** [review]

File: docs/upgrade/v1-3-2-to-v1-4-0.md [hunk 1]
```diff
@@ -32,3 +32,53 @@ When upgrading from v1.3.2 to v1.4.0, the upgrade process may become stuck if a
 For more information, see [Issue #7005](https://github.com/harvester/harvester/issues/7005).
 
 ---
+
+### 2. Upgrade stuck on waiting harvester bundle
+
+When upgrading from v1.3.2 to v1.4.0, the upgrade process may become stuck on waiting for the harvester bundle to be ready. This issue is caused by race condition when the fleet-agent is redeployed.
+
+If you encounter the issue, you will get the following error state:
```
Comment (jillian-maroket) [inline docs/upgrade/v1-3-2-to-v1-4-0.md:40]:
```suggestion
### 2. Upgrade stuck on waiting for Harvester bundle

When upgrading from v1.3.2 to v1.4.0, the upgrade process may become stuck on waiting for the Harvester bundle to become ready. This issue is caused by a race condition when the Fleet agent (`fleet-agent`) is redeployed.

The following error messages indicate that the issue exists.
```

File: docs/upgrade/v1-3-2-to-v1-4-0.md [hunk 2]
```diff
@@ -32,3 +32,53 @@ When upgrading from v1.3.2 to v1.4.0, the upgrade process may become stuck if a
...
+
+```shell
+> kubectl get bundles -n fleet-local
+NAME                                          BUNDLEDEPLOYMENTS-READY   STATUS
+mcc-harvester                                 0/1                       ErrApplied(1) [Cluster fleet-local/local: encountered 2 deletion errors. First is: admission webhook "validator.harvesterhci.io" denied the request: Internal error occurred: no route match found for DELETE /v1, Kind=Secret harvester-system/sh.helm.release.v1.harvester.v2]
+mcc-harvester-crd                             0/1                       ErrApplied(1) [Cluster fleet-local/local: admission webhook "validator.harvesterhci.io" denied the request: Internal error occurred: no route match found for DELETE /v1, Kind=Secret harvester-system/sh.helm.release.v1.harvester-crd.v1]
+```
+
+You can apply following script to fix the issue:
```
Comment (jillian-maroket) [inline docs/upgrade/v1-3-2-to-v1-4-0.md:49]:
```suggestion
You can run the following script to fix the issue.
```

**Review by jillian-maroket — COMMENTED** [review]
Just a few nits

**Comment by FrankYang0529** [comment]
Hi @jillian-maroket, thanks for review. I updated all pages.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**